libs/indibase/inditelescope.cpp

```diff
@@ -104,7 +104,7 @@ bool Telescope::initProperties()
     IUFillTextVector(&TimeTP, TimeT, 2, getDeviceName(), "TIME_UTC", "UTC", SITE_TAB, IP_RW, 60, IPS_IDLE);
 
     IUFillNumber(&LocationN[LOCATION_LATITUDE], "LAT", "Lat (dd:mm:ss.s)", "%012.8m", -90, 90, 0, 0.0);
-    IUFillNumber(&LocationN[LOCATION_LONGITUDE], "LONG", "Lon (dd:mm:ss.s)", "%012.8m", -180, 180, 0, 0.0);
+    IUFillNumber(&LocationN[LOCATION_LONGITUDE], "LONG", "Lon (dd:mm:ss.s)", "%012.8m", -180, 360, 0, 0.0);
```

Comment:
This cannot be used as well, it's 0 to 360 as the standard properties specify. All drivers follow this convention.

Reply:
OK, set back to 0-360 which is what it was.